report/21_Process-Developer-Interaction.md

```diff
@@ -1 +1,5 @@
-How do you interact as developers?
+The developers interact mainly through a weekly meeting right after the lecture, through the use of a Discord Server, and through the use of the GitHub Repository on which the project is hosted. After the lockdown caused be Covid-19, the weekly meeting started being held over Discord as well. 
+Discord has been used for direct communication, both in text-channels for quick questions and requests, and through voice channels for meetings or collaborative work.
+GitHub has been used to track work throughout the project, mainly via Issues assigned to developers. Some communication is also done via GitHub in the form of comments on pull requests. 
```

Comment:
I would elaborate on this point about github issues as it was a major part of the workflow. Feedback was also mainly contained to the Pull Request so that they were easier to read and all information was contained there.